sync_test.go

```diff
@@ -49,11 +55,6 @@ func TestFsSyncer_Sync(t *testing.T) {
 			expectedChanges: []string{},
 			syncOptions:     []func(*FsSyncer){WithChecksum},
 		},
-		"it should not replace a file with the same size and mtime": {
-			fixtureSrc:      "src/file",
-			fixtureDst:      "dst/rsync-file",
-			expectedChanges: []string{},
-		},
```

Comment:
Just like in commit b0bb006de1172f72a53e99b2cd47a36bf2f184f9, I had to remove this test as it kept failing. It cannot work. It is based on the fact that the mtime of both `src/file` and `dst/rsync-file` are the same but that is currently not guaranteed by the test. The mtime of these files corresponds to the time of when the `git clone` created the files locally. 

I updated PR #8 to restore this test after we thought about how to properly write these tests.